apps/time/src/widgets/home/ui/HomeCard.tsx

```diff
@@ -4,10 +4,9 @@ import { ComponentPropsWithoutRef, forwardRef } from 'react';
 
 import { cn } from '@clab/utils';
 
+import { StrictPropsWithChildren } from '@/shared/types';
```

Comment:
```suggestion
import type { StrictPropsWithChildren } from '@/shared/types';
```
type을 import할 때 import type을 사용하는 것이 좋습니다.

- https://velog.io/@hamjw0122/TS-import-type%EC%9D%84-%EC%82%AC%EC%9A%A9%ED%95%B4%EC%95%BC-%ED%95%98%EB%8A%94-%EC%9D%B4%EC%9C%A0